backend/src/main/java/com/znane_groby/backend/controller/GraveyardController.java

```diff
@@ -0,0 +1,102 @@
+package com.znane_groby.backend.controller;
+
+import com.znane_groby.backend.model.Graveyard;
+import com.znane_groby.backend.repository.CityRepository;
+import com.znane_groby.backend.repository.GraveyardRepository;
+import com.znane_groby.backend.util.ItemNotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.hateoas.IanaLinkRelations;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@Slf4j
+@RestController
+public class GraveyardController {
+    private final GraveyardRepository repository;
+    private final CityRepository cityRepository;
+    private final GraveyardResourceAssembler assembler;
+
+    GraveyardController(GraveyardRepository repository, GraveyardResourceAssembler assembler,CityRepository cityRepository) {
```

Comment:
To nie powinno być `@Autowired`? Podpina się samo?

Reply:
nie wiem :) tak na pewno dziala. To dodać Autowired?